Tests/Source/RangeTester.cpp

```diff
@@ -3,24 +3,18 @@
 #include <catch.hpp>
 #include "Rover/Constant.hpp"
 #include "Rover/Generator.hpp"
+#include "Rover/Noncopyable.hpp"
 #include "Rover/Range.hpp"
+
 using namespace Rover;
 
 namespace {
 
   template<typename T>
-  class MoveOnlyConstant : public Constant<T> {
+  class MoveOnlyConstant : public Constant<T>, public Noncopyable {
     public:
 
```

Comment:
Remove blank line on 16.

Reply:
A bit lost with public interfaces too. We tend to put a blank line after public: in API/documented classes, but we don't do so for type traits and apparently some internal classes as well. Where is the boundary?